### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: static/js/src/scores.js [hunk 1]
```diff
@@ -16,18 +16,21 @@ function scores() {
                 postScore = Number(postScore);
 
                 var learningObjectiveScore = postScore - preScore;
-                var tableRow = $('<tr></tr>');
+                var tableRow = $('<tr id="' + prefix + '"></tr>');
```
Comment (nikolas) [inline static/js/src/scores.js]: HTML id's should be unique.. and seems here a lot of this tr and td code would make duplicate IDs, but maybe I'm wrong? Anyways, the id attr should be change to class, or data attributes, if there are duplicate ids.

Comment (nbuonin) [reply static/js/src/scores.js]: These should always be unique. The prefix is built up from the index of the map on line 8.

Comment (nikolas) [reply static/js/src/scores.js]: Oh I see, cool.

Comment (nbuonin) [reply static/js/src/scores.js]: I'm using the `[...Array(3).keys]` as a range function, but its confusing, I could make that more clear.

**Pull request merged**